DTAConfig/OptionPanels/DisplayOptionsPanel.cs

```diff
@@ -1,17 +1,21 @@
+using Localization;
 using ClientCore;
 using ClientGUI;
-using Localization;
-using Microsoft.Win32;
 using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
 using Rampastring.Tools;
 using Rampastring.XNAUI;
 using Rampastring.XNAUI.XNAControls;
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
-using System.IO;
+using System.Runtime.InteropServices;
+#if WINFORMS
 using System.Windows.Forms;
+#endif
+#if TS
+using Microsoft.Win32;
```

Comment:
Um pretty sure this won't work on Linux?

Reply:
True, all relevant code is guarded by this check:

```C# 
RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
```

So it works as expected.

Reply:
I meant wouldn't this using result in an error on Linux?

Reply:
No it works as expected, I tested TS on fedora.